### PR TITLE
feat: bunny cdn integrations

### PR DESCRIPTION
**Pull request by csosamorales**

### CODE REVIEW

File: ui/src/store/features/bunny-cdn/bunny-cdn-client.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { env } from '@/constants';
+import { AppLog } from '@/utils';
+import axios from 'axios';
+import * as crypto from 'crypto';
+
+const client = (body: string, endpoint: string) => {
+  if (!env.bunnyCDN.feSigningKey) {
+    AppLog.error('Missing BunnyCDN signing key');
+    throw new Error();
```
Comment (zoruka) [inline ui/src/store/features/bunny-cdn/bunny-cdn-client.ts:9]: should we add a message to all these errors?

File: ui/src/store/features/bunny-cdn/bunny-cdn-client.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+const createPullzone = async (sourceDomain: string, targetDomain: string) => {
+  try {
+    const body = JSON.stringify({
+      sourceDomain,
+      targetDomain,
+    });
+
+    if (!env.bunnyCDN.createPullzone) {
+      AppLog.error('Missing BunnyCDN create pullzone endpoint');
+      throw new Error();
```
Comment (zoruka) [inline ui/src/store/features/bunny-cdn/bunny-cdn-client.ts:38]: same here

File: ui/src/store/features/bunny-cdn/bunny-cdn-client.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+const verifyPullzone = async (hostName: string) => {
+  try {
+    const body = JSON.stringify({
+      hostName,
+    });
+
+    if (!env.bunnyCDN.verifyPullzone) {
+      AppLog.error('Missing BunnyCDN verify pullzone endpoint');
+      throw new Error();
```
Comment (zoruka) [inline ui/src/store/features/bunny-cdn/bunny-cdn-client.ts:57]: same here

File: ui/src/store/features/bunny-cdn/bunny-cdn-client.ts (hunk 1 above)
Comment (csosamorales) [reply ui/src/store/features/bunny-cdn/bunny-cdn-client.ts:9]: gotcha

**Review by zoruka — APPROVED** [review]

**Pull request merged**